.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: Build
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
main or master

Reply:
added master to it (future compatibility :-)). Thank you!